### DISCUSSION TITLE
Keycloak 19.0.2 Wildfly version?

### DISCUSSION THREAD

**Discussion by bonnm** [keycloak/keycloak#14406] [Q&A] [answered]
Hi,
Ok, so Keycloak 19.0.2 is out, but in contrast to 19.0.1, there's no downloadable Wildfly version. Is it coming later? As the milestones suggest, Wildfly version will removed in version 20, not earlier: [https://github.com/keycloak/keycloak/issues/13855](https://github.com/keycloak/keycloak/issues/13855)

**Comment by darius-m** [comment]
I think the GitHub actions that build the WildFly version's archives, documentation, tests were removed and the changes were included before this version was published - see #13367.

**Comment by stianst** [comment, marked as answer]
Fixed for 19.0.2 - this is caused by a small issue with the website where adding a new micro version uses the download template for the next major/minor instead of the current major/minor.

**Reply by bonnm** [reply]
Thank you!
